.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -8,106 +8,110 @@ assignees: ''
-        * Change the version
+    * Check that the version of the `nydus-snapshotter` used by Kata matches the one used by the operator
+        * Compare `nydus-snapshotter` version in Kata [versions.yaml](https://github.com/kata-containers/kata-containers/blob/main/versions.yaml#L291) [this]() with the [Makefile](https://github.com/confidential-containers/operator/blob/main/install/pre-install-payload/Makefile#L4) for the opertor pre-intall container.
+            * If they do not match, update the operator. This can be part of the PR described in the next step.
+    
+    * Make sure that the operator pulls the most recent version of the pre-install container
+        * Find the last commit in the [pre-install directory](https://github.com/confidential-containers/operator/tree/main/install/pre-install-payload)
+            * Make sure that the commit matches the preInstall / postUninstall image specified for [enclave-cc CRD](https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml) and [Kata CRD](https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/default/kustomization.yaml)
+                * If these do not match (for instance if you changed the snapshotter above), update the operator so that they do match. This can be part of the PR described in the next step.
+    
 
-- [ ] 3. Wait for kata-runtime-payload-ci to be successfully built
-    * After the previous PR is merged wait for the kata-runtime-payload-ci (https://github.com/kata-containers/kata-containers/actions/workflows/cc-payload-after-push.yaml) has completed, so the latest kata-runtime-payload-ci contains the changes
+- [ ] 4. :wrench: **Open a PR to the operator to update the release artifacts**
 
-- [ ] 4. Check if there are new changes in the pre install payload script
+    Update the operator to use the payloads identified in steps 1 and 2.
 
-    * https://github.com/confidential-containers/operator/tree/main/install/pre-install-payload
-      * The last commit there must match what's in the following files as preInstall / postUninstall image
-        * Enclave CC: https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml
-        * Kata Containers:
-              Note that for Kata Containers, we're looking for the newTag, below the quay.io/confidential-containers/reqs-payload image
-          * default: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/default/kustomization.yaml
+    For enclave-cc, the payload should have been generated [here](quay.io/confidential-containers/runtime-payload-ci)
 
-- [ ] 5. Ensure the Operator is using the latest CI builds and that the Operator tests are passing
+    For Kata, the payload should be [here](https://github.com/kata-containers/kata-containers/pkgs/container/kata-deploy-ci/versions?filters%5Bversion_type%5D=tagged).
```

Comment:
I guess if we are using a kata release (as per step 2) we should get it from https://quay.io/repository/kata-containers/kata-deploy?tab=tags?

Reply:
Yes, I gotta update all the locations for the images.